entry_types/scrolled/package/src/frontend/AtmoAudio.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import {media} from 'pageflow/frontend';
+
+import { useSectionLifecycle } from './useSectionLifecycle';
+import {useFile} from '../entryState';
+import {processSources} from './AudioPlayer';
+import { useAtmo } from './useAtmo';
+
+/**
+ * Plays audio file in background
+ *
+ * @param {Object} props
+ * @param {number} props.atmoAudioFileId - Perma id of the audio file.
+ *
+ */
+export function AtmoAudio(props) {
+  const audioFile = useFile({collectionName: 'audioFiles', permaId: props.atmoAudioFileId});
+  let atmo = useAtmo();
+  
+  useSectionLifecycle({
+    onActivate(){
+      let sources = undefined;
+      if (audioFile && audioFile.isReady) {
+        sources = processSources(audioFile);
+      }
+      
+      atmo.updateAtmo({
+        sources: sources,
+        atmoAudioFileId: props.atmoAudioFileId
+      });
+    }
+  });
```

Comment:
Atmo audio also needs to be updated when the file changes. Currently selecting an atmo audio file does not start the audio. Only when I deactivate and activate the section again the atmo starts.

You can use a combination of `usePrevious` and an effect that compares audio files to also trigger `updateAtmo` when the atmo changes.